content/en/_index.Rmd

```diff
@@ -353,7 +362,7 @@ fit <- fit_seir(
   e_prior = c(0.85, 0.05), # 85% of residents social distancing
   start_decline_prior = c(log(24), 0.1), # Starting date of closing of non-essential businesses (March 24th, 2020)
   end_decline_prior = c(log(34), 0.1), # 10 day buffer to ramp in social distancing
-  f_prior = cbind(c(0.4, 0.5, 0.6), c(0.2, 0.2, 0.2)),
+  f_prior = cbind(c(0.4, 0.5, 0.6, 0.65, 0.6, 0.6), c(0.2, 0.2, 0.2, 0.2, 0.2)),
```

Comment:
@DougManuel could you have a look at this parameter? This parameter generates a vector of contact rate fractions (f) with their respective standard deviations.

In the `covidseir` documentation they use a vector of `c(0.4, 0.5, 0.6)` for the f values during lockdown, after 1 reopening, and after a 2nd reopening. Please have a look at the additional f values added for phase 3, modified phase 2 rollback, and the ending of the modified phase 2 rollback.

Reply:
Let's talk